teku/src/main/java/tech/pegasys/teku/cli/options/P2POptions.java

```diff
@@ -150,11 +150,19 @@ public String getP2pPrivateKeyFile() {
   }
 
   public int getP2pLowerBound() {
-    return p2pLowerBound;
+    if (p2pLowerBound > p2pUpperBound) {
+      return p2pUpperBound;
+    } else {
+      return p2pLowerBound;
+    }
```

Comment:
Math.min?

Reply:
I just added the warnings and with those warnings embedded in the body of the if statements, I don't know how I can incorporate Math.min and max.

```  
public int getP2pLowerBound() {
    if (p2pLowerBound > p2pUpperBound) {
      LOG.warn("Adjusting target number of peers lower bound to equal upper bound, which is {}", p2pUpperBound);
      return p2pUpperBound;
    } else {
      return p2pLowerBound;
    }
  }
```